SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -252,11 +263,25 @@ void USpatialGameInstance::OnLevelInitializedNetworkActors(ULevel* LoadedLevel,
 		return;
 	}
 
+	if (SpatialConnectionManager->IsConnected())
```

Comment:
Once I switch to the LBStrategy, I'll need a different check here, but for now, this is enough.